### PR TITLE
Userinfo and Serverinfo: Add some clarity to Username and date/time

### PR DESCRIPTION
**Pull request by Shelme74**
Userinfo:

- Nickname and Username+discriminant have been put on the same line
- Date and times have been set to UTC, and full letters months have been added for clarity

Serverinfo:

- Date and times have been set to UTC, and full letters months have been added for clarity

image

### CODE REVIEW

File: Discord.Addons.Interactive/Discord.Addons.Interactive.csproj [hunk 1]
```diff
@@ -23,6 +23,10 @@
     
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
```
Comment (iShauny) [inline Discord.Addons.Interactive/Discord.Addons.Interactive.csproj:28]: is this intentional?

File: Floofbot/Floofbot.csproj [hunk 2]
```diff
@@ -5,20 +5,24 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
+
```
Comment (iShauny) [inline Floofbot/Floofbot.csproj:11]: same comment as before, although not against updating but think it should be in a future PR

File: Floofbot/Modules/Utilities.cs [hunk 3]
```diff
@@ -49,16 +49,20 @@ public async Task UserInfo(IGuildUser usr = null)
...
 
+            // Get user's Discord joining date and time, in UTC
+            string discordJoin = user.CreatedAt.ToUniversalTime().ToString("dd\\/MMM\\/yyyy \\a\\t H:MM \\U\\T\\C");
+            // Get user's Guild joining date and time, in UTC
+            string guildJoin = user.JoinedAt?.ToUniversalTime().ToString("dd\\/MMM\\/yyyy \\a\\t H:MM \\U\\T\\C");
+
             if (user.AvatarId != null)
                 avatar = user.GetAvatarUrl(ImageFormat.Auto, 512);
 
-            string infostring = $"👥 **User info for:** {user.Mention}\n";
```
Comment (iShauny) [inline Floofbot/Modules/Utilities.cs:55]: I think I personally prefer it to say the old version than the new one here

**Review by iShauny — CHANGES_REQUESTED** [review]

File: Discord.Addons.Interactive/Discord.Addons.Interactive.csproj (hunk 1 above)
Comment (Shelme74) [reply Discord.Addons.Interactive/Discord.Addons.Interactive.csproj:28]: It is, so we'll also have debug info in Visual Studio's console.

File: Floofbot/Floofbot.csproj (hunk 2 above)
Comment (Shelme74) [reply Floofbot/Floofbot.csproj:11]: I see. I'll set those back to their previous values.

File: Floofbot/Modules/Utilities.cs (hunk 3 above)
Comment (Shelme74) [reply Floofbot/Modules/Utilities.cs:55]: Yes, it'll be more clear that way

**Review by iShauny — APPROVED** [review]

**Pull request merged**